leveldb/external_test.go

```diff
@@ -10,8 +10,8 @@ import (
 	. "github.com/onsi/ginkgo"
```

Comment:
If it will be pulling down from master, I'd rather we fork them so we can control what is getting pulled down even though they're only used for tests. If we can specify a git hash to pull, then we can leave them upstream.